src/manage_pg.c

```diff
@@ -212,17 +212,79 @@ manage_create_sql_functions ()
        " LANGUAGE C;",
        GVM_LIB_INSTALL_DIR);
 
-  sql ("CREATE OR REPLACE FUNCTION level_max_severity (text, text)"
-       " RETURNS double precision"
-       " AS '%s/libgvm-pg-server', 'sql_level_max_severity'"
-       " LANGUAGE C;",
-       GVM_LIB_INSTALL_DIR);
+  sql ("CREATE OR REPLACE FUNCTION level_max_severity (lvl text, cls text)"
+       "RETURNS double precision AS $$"
+       "DECLARE"
+       "  v double precision;"
+       "BEGIN"
+       "  CASE"
+       "    WHEN lvl = 'Log' THEN"
+       "      v := 0.0;" // SEVERITY_LOG
```

Comment:
How about using the actual definition, in case they change?  Something like
```suggestion
       "      v := " G_STRINGIFY (SEVERITY_LOG) ";"
```

Reply:
Done that in commit 01a6cba86195ac958d504a3805b51940cadcf9e1